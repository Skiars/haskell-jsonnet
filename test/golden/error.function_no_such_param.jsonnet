local id(a) = a;
id("foo", b = 2)
