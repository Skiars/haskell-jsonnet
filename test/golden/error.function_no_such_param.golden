Runtime error: function has no parameter 'b'
    ./test/golden/error.function_no_such_param.jsonnet:1:1,3:1